lib/meandro/rules/unused_callbacks.ex

```diff
@@ -34,33 +36,35 @@ defmodule Meandro.Rule.UnusedCallbacks do
   end
 
   defp analyze_module(file, ast) do
-    {_, acc} = Macro.prewalk(ast, %{current_module: nil, callbacks: []}, &collect_callbacks/2)
+    {_, acc} = Macro.prewalk(ast, %{module: nil, callbacks: []}, &collect_callbacks/2)
     %{callbacks: callbacks} = acc
 
-    for {module, name, arity, line, count} <- callbacks, count == 0 do
+    for {module, name, arity, line, count} <- Enum.reverse(callbacks), count == 0 do
       %Meandro.Rule{
         file: file,
         line: line,
         text: "Callback #{module}:#{name}/#{arity} is not used anywhere in the module",
-        pattern: {name, arity}
+        pattern: {module, name, arity}
```

Comment:
Do we need to add `module` to all our ignore patterns? I would expect that to be taken care of externally. That is… I would expect whatever we implement in #33 to first filter by module on its own and then only call `is_ignored?/2` with patterns and keys that belong to the same module.

Reply:
Yeah that makes sense :)